### PR TITLE
Use variable delta time for velocity

### PR DESCRIPTION
**Pull request by OliverW10**

### CODE REVIEW

**Review by james-ward — APPROVED** [review]
lgtm

File: components/chassis.py [hunk 1]
```diff
@@ -260,23 +261,27 @@ def execute(self) -> None:
 
         self.update_odometry()
 
+        dt = min(time.monotonic() - self.last_time, 0.1)
```
[the remark this answers is not in the document]
Comment (OliverW10) [reply components/chassis.py:264]: I capped it at 100ms as a safeguard against getting a very high dt when the robot first enables.

[the remark this answers is not in the document]
Comment (auscompgeek) [reply components/chassis.py:264]: Ah, makes sense. Would it be a good idea to instead update the last time in on_enable?

**Review by auscompgeek — APPROVED** [review]
lgtm with nit

**Pull request merged**